specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/instancePools.json

```diff
@@ -0,0 +1,453 @@
+            "in": "path",
+            "description": "The name of the instance pool to be updated.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The requested instance pool resource state.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/InstancePoolUpdate"
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated the instance pool.",
+            "schema": {
+              "$ref": "#/definitions/InstancePool"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidResourceRequestBody - The resource or resource properties in the request body is empty or invalid."
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Patch an instance pool": {
+            "$ref": "./examples/PatchInstancePool.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/instancePools": {
+      "get": {
+        "tags": [
+          "InstancePools"
+        ],
+        "description": "Gets a list of instance pools in the resource group",
+        "operationId": "InstancePools_ListByResourceGroup",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the list of instance pools.",
+            "schema": {
+              "$ref": "#/definitions/InstancePoolListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List instance pools by resource group": {
+            "$ref": "./examples/ListInstancePools.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "InstancePoolProperties": {
+      "description": "Properties of an instance pool.",
+      "required": [
+        "subnetId",
+        "vCores",
+        "licenseType"
+      ],
+      "type": "object",
+      "properties": {
+        "subnetId": {
+          "description": "Resource ID of the subnet to place this instance pool in.",
```

Comment:
I assume you will be adding a linked access check for this property?

Reply:
Yup, we will be adding a linked access check for subnet in ARM Manifest.